environment.yml

```diff
@@ -6,6 +6,7 @@ dependencies:
   - conda-forge::ipywidgets=7.4.2
   - conda-forge::matplotlib=3.0.3
   - conda-forge::nb_conda_kernels=2.2.1
+  - conda-forge::nbconvert=5.6.1
```

Comment:
Here's the traceback from the error that was occurring due to this unpinned implicit dependency:

```python-traceback
[E 06:34:34.698 NotebookApp] Uncaught exception GET /notebooks/nb/1.hetionet_computation/1.compute_priors_and_sampled_priors.ipynb (::1)
    HTTPServerRequest(protocol='http', host='localhost:8889', method='GET', uri='/notebooks/nb/1.hetionet_computation/1.compute_priors_and_sampled_priors.ipynb', version='HTTP/1.1', remote_ip='::1')
    Traceback (most recent call last):
      File "/home/dhimmel/miniconda3/envs/xswap-analysis/lib/python3.7/site-packages/tornado/web.py", line 1711, in _execute
        result = method(*self.path_args, **self.path_kwargs)
      File "/home/dhimmel/miniconda3/envs/xswap-analysis/lib/python3.7/site-packages/tornado/web.py", line 3208, in wrapper
        return method(self, *args, **kwargs)
      File "/home/dhimmel/miniconda3/envs/xswap-analysis/lib/python3.7/site-packages/notebook/notebook/handlers.py", line 59, in get
        get_custom_frontend_exporters=get_custom_frontend_exporters
      File "/home/dhimmel/miniconda3/envs/xswap-analysis/lib/python3.7/site-packages/notebook/base/handlers.py", line 519, in render_template
        return template.render(**ns)
      File "/home/dhimmel/miniconda3/envs/xswap-analysis/lib/python3.7/site-packages/jinja2/environment.py", line 1301, in render
        self.environment.handle_exception()
      File "/home/dhimmel/miniconda3/envs/xswap-analysis/lib/python3.7/site-packages/jinja2/environment.py", line 936, in handle_exception
        raise rewrite_traceback_stack(source=source)
      File "/home/dhimmel/miniconda3/envs/xswap-analysis/lib/python3.7/site-packages/notebook/templates/notebook.html", line 1, in top-level template code
        {% extends "page.html" %}
      File "/home/dhimmel/miniconda3/envs/xswap-analysis/lib/python3.7/site-packages/notebook/templates/page.html", line 154, in top-level template code
        {% block header %}
      File "/home/dhimmel/miniconda3/envs/xswap-analysis/lib/python3.7/site-packages/notebook/templates/notebook.html", line 120, in block 'header'
        {% for exporter in get_custom_frontend_exporters() %}
      File "/home/dhimmel/miniconda3/envs/xswap-analysis/lib/python3.7/site-packages/notebook/notebook/handlers.py", line 19, in get_custom_frontend_exporters
        from nbconvert.exporters.base import get_export_names, get_exporter
      File "/home/dhimmel/miniconda3/envs/xswap-analysis/lib/python3.7/site-packages/nbconvert/__init__.py", line 4, in <module>
        from .exporters import *
      File "/home/dhimmel/miniconda3/envs/xswap-analysis/lib/python3.7/site-packages/nbconvert/exporters/__init__.py", line 3, in <module>
        from .html import HTMLExporter
      File "/home/dhimmel/miniconda3/envs/xswap-analysis/lib/python3.7/site-packages/nbconvert/exporters/html.py", line 14, in <module>
        from jinja2 import contextfilter
    ImportError: cannot import name 'contextfilter' from 'jinja2' (/home/dhimmel/miniconda3/envs/xswap-analysis/lib/python3.7/site-packages/jinja2/__init__.py)
```